packages/core/useRefHistory/index.ts

```diff
@@ -42,6 +42,10 @@ export interface UseRefHistoryOptions<Raw, Serialized = Raw> {
    * Deserialize data from the history
    */
   parse?: (v: Serialized) => Raw
+  /**
+   * Can get throttleFilter and debounceFilter
+   */
+  filter?: EventFilter
```

Comment:
This should be called `eventFilter`, no?

Reply:
Every other options is only one word, so eventFilter will contrast with them. But if you want, I can change it.

Reply:
`filter` is a good name looking only at `useRefHistory`, and if I recall correctly it was the name we used at one point but it was later replaced by `eventFilter` everywhere because it was more explicit. You can see the interface used in other composables (like `useStorage`, `useMouse`, ...) here https://github.com/vueuse/vueuse/blob/d6dd1ba2d5fab674771103f0955e631c6c8f436e/packages/shared/utils/filters.ts#L17
So it is better that we align the naming for the event filter with the rest of VueUse.

Reply:
Naming is the hardest part of programming😃

Reply:
Indeed 🙂. At least in this case the choice was already made by Anthony 😅

Reply:
Anthony also wrote me this declaration
`options: Omit<UseRefHistoryOptions<Raw, Serialized>, **'filter'**> & { throttle? MaybeRef<number> }`
so it confused me

Reply:
@patak-js updated

Reply:
Yeah, my bad. Would like to have `eventFilter` for it.